spec/features/applications/new_application_spec.rb

```diff
@@ -14,7 +14,7 @@
         description: "Nice dog"
         )
       @pet_2 = @shelter_1.pets.create!(
-        image: 'https://s3.amazonaws.com/cdn-origin-etr.akc.org/wp-content/uploads/2017/11/12234558/Chinook-On-White-03.jpg',
+        image: 'https://petsfeed.co/wp-content/uploads/2019/04/German-Shepherd-dog.jpg',
```

Comment:
Only issue when we do this could be if the pet is using a default image and so is another favorited pet. Default images were part of an earlier user story...something to consider if we have time at the end.